src/semantic-ui/site/views/item.overrides

```diff
@@ -131,7 +125,7 @@ Item Overrides - REACT-INVENIO-APP-ILS
       }
 
       &.bkg-yellow {
-        background-color: #fff4bd !important;
+        background-color: ##ffffe6 !important;
```

Comment:
does it affect any other places in the application ?

Reply:
No